test/Parse/unclosed-string-interpolation.swift

```diff
@@ -2,14 +2,26 @@
 
 let mid = "pete"
 
-_ = "mid == \(pete" // expected-error {{expected ')' at end of string interpolation}} expected-note {{to match this opening '('}}
-let theGoat = "kanye \(" // expected-error {{expected ')' at end of string interpolation}} expected-note {{to match this opening '('}}
-let equation1 = "2 + 2 = \(2 + 2" // expected-error {{expected ')' at end of string interpolation}} expected-note {{to match this opening '('}}
+_ = "mid == \(pete" 
+// expected-error @-1 {{expected ')' at end of string interpolation}} expected-note @-1 {{to match this opening '('}}
+
+let theGoat = "kanye \("
+// expected-error @-1 {{expected ')' at end of string interpolation}} expected-note @-1 {{to match this opening '('}}
+
+let equation1 = "2 + 2 = \(2 + 2" 
+// expected-error @-1 {{expected ')' at end of string interpolation}} expected-note @-1 {{to match this opening '('}}
 
 _ = """
 \(
 """
 // expected-error @-2 {{expected ')' at end of string interpolation}} expected-note @-2 {{to match this opening '('}}
```

Comment:
This one will nullify subsequent tests useless you move it to the bottom.

Reply:
There